### ISSUE TITLE
lighty-community-restconf-netconf-app remote access

### ISSUE THREAD

**Issue by maurizio-lazzarini** [PANTHEONtech/lighty#1957]
After successful installation and start of lighty-community-restconf-netconf-app-20.1.0 no problem to execute the  http://localhost:8888/restconf/operations GET request.

Is it possible to transmit the GET from a remote PC ?
I have tried http://<IP address>:8888/restconf/operations from a remote PC, where <IP address> is the address of the PC running the lighty-community-restconf-netconf app but no connection was possible.

I suppose this is due to the fact that the port 8888 is configured only for local access (sudo netstat -nltp returns: 127.0.0.1:8888).
How to change tha app to allow remote access ?

**Comment by ihrasko** [comment]
Yes this can be configured here: https://github.com/PANTHEONtech/lighty/blob/main/lighty-examples/lighty-community-restconf-netconf-app/src/main/assembly/resources/sampleConfigSingleNode.json#L51.

By default it binds localhost only.

Best,

**Comment by maurizio-lazzarini** [comment]
I modified the indicated file by substituting the inetAddress value from 127.0.0.1 to the static IP address of the VirtualBox machine running UBUNTU on which lighty-community-restconf-netconf-app was installed. 
After the app restart the access is still  possible via a local browser,  but not from a browser installed on the Windows OS on which VirtualBox is installed.  
The behaviour is the same I experiences before modifying the configuration file.

**Comment by ihrasko** [comment]
Unpack application:
```
cd lighty/lighty-examples/lighty-community-restconf-netconf-app/target
unzip lighty-community-restconf-netconf-app-20.3.0-bin.zip
cd lighty-community-restconf-netconf-app-20.3.0
nano sampleConfigSingleNode.json
```
Set from `"inetAddress": "127.0.0.1",` to `"inetAddress": "0.0.0.0",`.

Run:
```
java -jar lighty-community-restconf-netconf-app-20.3.0.jar sampleConfigSingleNode.json
```
Note:
Please use address of your VM instead of `0.0.0.0`.

Hope that helps!
